### DISCUSSION TITLE
I cant demonstrate that Im enrrolled in my college

### DISCUSSION THREAD

**Discussion by VirtueDevs** [community/community#79275] [GitHub Education]
### Hi everyone,

GitHub ask for physical documents to demonstrate my enrrollment, but in my college all documents that they provide are virtual, like the payment receipt or my  class schedule. What can I do?

**Comment by Cypher-PB** [comment]
**Contact your school** and they will provide the required document. If not you can use your virtual document

**Comment by manik-007** [comment]
Hello @VirtueDevs  

Maybe you can just print out the payment receipt and clicked it's picture and try uploading it if you don't have any physical documents like: id card, library card or somethng.

Just turn the virtual ones into physical but make sure that on it your name, course, duration of the course and institution name is written.
If you don't have a couple of the above things mentioned on the document then dont hesitate, just upload. It won't affect much.

Tell me if you get your developers pack using this trick and if have any doubt then feel free to ask i am here to answer :)
I hope this helped you.
